domain/src/main/java/com/globant/domain/entity/MemeDetailRoom.kt

```diff
@@ -0,0 +1,15 @@
+package com.globant.domain.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "memeDetail_table")
+class MemeDetailRoom(
+    @PrimaryKey @ColumnInfo(name = "id") val id: Int,
+    @ColumnInfo(name = "origin") val origin: String,
```

Comment:
think this @ColumIngfo tag its not necessary if your field has the same name of the column